src/services/meta_transaction_service.ts

```diff
@@ -113,6 +119,13 @@ export class MetaTransactionService {
     ): Promise<MetaTransactionQuoteResult> {
         const wrappedNativeToken = NATIVE_FEE_TOKEN_BY_CHAIN_ID[CHAIN_ID];
 
+        const affiliateFee: AffiliateFee = {
+            feeType: AffiliateFeeType.GaslessFee,
+            // TODO: what address do we need to use?
+            recipient: FEE_RECIPIENT_ADDRESS,
+            sellTokenPercentageFee: 0,
```

Comment:
We probably need better `AffiliateFee` types since neither of these fields are useful here.